### PR TITLE
NMS-10403: Fixed script file reloading

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://issues.opennms.org/browse/NMS-10403

### CODE REVIEW

File: features/telemetry/adapters/collection/src/main/java/org/opennms/netmgt/telemetry/adapters/collection/AbstractPersistingAdapter.java [hunk 1]
```diff
@@ -264,7 +263,7 @@ protected ScriptedCollectionSetBuilder getCollectionBuilder() throws Exception {
         if (builder == null) {
             // script didn't compile, set flag to false
             scriptCompiled.set(false);
-            throw new Exception(String.format("Error compiling script '%s'. See logs for details.", script));
```
Comment (mvrueden) [inline features/telemetry/adapters/collection/src/main/java/org/opennms/netmgt/telemetry/adapters/collection/AbstractPersistingAdapter.java]: Is it worth logging this information?

Comment (fooker) [reply features/telemetry/adapters/collection/src/main/java/org/opennms/netmgt/telemetry/adapters/collection/AbstractPersistingAdapter.java]: It's duplicated. The caller logs this in all cases. (i.e.: https://github.com/OpenNMS/opennms/blob/19d62fc7719042ad2c3ba6605486860ddf8ea17b/features/telemetry/adapters/nxos/src/main/java/org/opennms/netmgt/telemetry/adapters/nxos/NxosGpbAdapter.java#L135)

**Comment by j-white** [comment]
Can we re-target this to `release-23.0.1`? I believe the affected code is in 23.0.0.

**Comment by christianpape** [comment]
done

**Pull request merged**